dev/AccessControl/Security.AccessControl.h

```diff
@@ -0,0 +1,19 @@
+﻿#pragma once
+
+#include <Windows.h>
+
+struct AppContainerNameAndAccess
+{
+    PCWSTR appContainerName;
```

Comment:
PCWSTR or PWSTR?

Is the const-ness in the attribute a problem? I've seen compile problems from this sort of thing in the past

Reply:
Definitely `PCWSTR`. You can change the value of the pointer, but not what it points to. The const-ness is important if you ever want to assign a const string or literal to that field, aka `L"AccessControlTestAppPackage_8wekyb3d8bbwe"`.